tests/e2e/test_rosacli_kubelet_config.go

```diff
@@ -9,18 +11,17 @@ import (
 	"github.com/openshift/rosa/tests/utils/exec/rosacli"
 )
 
-var _ = Describe("Edit kubeletconfig",
+var _ = Describe("Kubeletconfig for classic cluster",
```

Comment:
nitpick if we want both features to be in sync :)
```suggestion
var _ = Describe("Kubeletconfig on Classic cluster",
```